client/my-sites/stats/all-time/index.jsx

```diff
@@ -67,7 +69,7 @@ class StatsAllTime extends Component {
 		return (
 			<div>
 				{ siteId && <QuerySiteStats siteId={ siteId } statType="stats" query={ query } /> }
-				<SectionHeader label={ translate( 'All-time posts, views, and visitors' ) } />
+				<SectionHeader label={ translate( 'All-time posts, comments, views, and visitors' ) } />
```

Comment:
:warning: This change will be queued for retranslation. We'll display the English text until then.